src/components/PrivateRoute.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Route, Redirect } from "react-router-dom";
+
+const PrivateRoute = ({ component: Component, ...rest }) => (
+  <Route
+    {...rest}
+    render={props =>
+      true ? (
+        <Component {...props} />
+      ) : (
+        <Redirect to={{ pathname: '/login' }} />
```

Comment:
You might want to start storing the routes in an object or sth so they can be easily accessed and they are easy to refactor if you later need to do so.